rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1,4 +1,4 @@
-#include <QButtonGroup>
+﻿#include <QButtonGroup>
```

Comment:
what's this.... We just moved back to UTF8.
Just use some other character like |  ..... no one cares if the ascii art is perfect

Reply:
In cpp file there isn't much difference BOM or not

Reply:
but it's annoying AF because now it will changed all the time if someone commits code.
Please just use some other solution

Reply:
Okay, if you insist, I'll remove BOM, just checked and VS seems to read it correctly nevertheless.